test/e2e_node/node_shutdown_linux_test.go

```diff
@@ -188,10 +194,10 @@ func getGracePeriodOverrideTestPod(name string, node string, gracePeriod int64,
 					Args: []string{`
 _term() {
 	echo "Caught SIGTERM signal!"
-	sleep infinity
+	while true; do sleep 5; done
```

Comment:
Another option would be to set it to sufficiently high value.

Reply:
ack, that should work as well. I went with `while true; sleep` loop since it more closely matches "infinity" sleep  and seems to be the common pattern in other tests (e.g. https://github.com/kubernetes/kubernetes/blob/bd2e557/test/e2e_node/eviction_test.go#L832).